### PR TITLE
Add 20200225.2 wrap

### PR DESCRIPTION
**Pull request by nnooney**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,356 @@
+project('abseil-cpp', 'cpp',
+  version: '20200225.2',
+  license: 'Apache-2.0')
+
+add_project_arguments('-Wno-sign-compare', language: 'cpp')
```
Comment (legeana) [inline meson.build]: Please use `get_supported_arguments` to filter out unsupported arguments.
https://mesonbuild.com/Release-notes-for-0-43-0.html#easier-handling-of-supported-compiler-arguments

**Review by legeana — CHANGES_REQUESTED** [review]

File: meson.build (hunk 1 above)
Comment (nnooney) [reply meson.build]: Done.

File: meson.build [hunk 2]
```diff
@@ -0,0 +1,364 @@
+project('abseil-cpp', 'cpp',
+  version: '20200225.2',
+  license: 'Apache-2.0')
+
+cc = meson.get_compiler('c')
```
Comment (legeana) [inline meson.build]: Please use `'cpp'` here.

File: meson.build [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+
+c_flags = [
```
Comment (legeana) [inline meson.build]: Optional: `cc_flags` or `cpp_flags` might be more appropriate.

**Review by legeana — CHANGES_REQUESTED** [review]

File: meson.build (hunk 2 above)
Comment (nnooney) [reply meson.build]: Done.

File: meson.build (hunk 3 above)
Comment (nnooney) [reply meson.build]: Done (cpp_flags).

**Pull request merged**